modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/A11yTool.testcase

```diff
@@ -77,6 +77,117 @@ definition {
 		takeScreenshot();
 	}
 
+	@description = "LPS-142329. Verifies 2 occurrences of the same violation are listed in the side panel."
+	@priority = "3"
+	test CanDetectMultipleOccurencesOfSameViolations {
+		property osgi.module.configuration.file.names = "com.liferay.frontend.js.a11y.web.internal.configuration.FFA11yConfiguration.config";
+		property osgi.module.configurations = "enable=B&quot;true&quot;";
+
+		// Workaround for side panel sometimes not displaying LPS-141442
+
+		while (IsElementNotPresent(locator1 = "A11yTool#VIOLATION_PANEL_HEADER")) {
+			Refresh();
+		}
+
+		task ("Verifies the panel is in top-level view") {
+			WaitForElementPresent(locator1 = "A11yTool#VIOLATION_PANEL_HEADER");
+
+			WaitForElementPresent(
+				key_text = "",
+				locator1 = "A11yTool#VIOLATION_PANEL_RULE_LIST_ITEM");
+		}
+
+		task ("Verify 2 of the same violation is on the page") {
+			Click(
+				key_index = "1",
+				locator1 = "A11yTool#HIGHLIGHTED_ELEMENT_INDEX");
+
+			Click(
+				key_index = "2",
+				locator1 = "A11yTool#HIGHLIGHTED_ELEMENT_INDEX");
+
+			WaitForElementPresent(
+				key_text = "button-name",
+				locator1 = "A11yTool#HIGHLIGHTED_ELEMENT_POPOVER_TITLE");
```

Comment:
After clicking the first and the second highlighted elements, there will be two popovers in the page, right?  But for my understanding, this WaitForElementPresent only wait for the first one, can we add a new path of the popover title with variable key_index, so that we can assert both the popovers?

Reply:
see https://github.com/liferay-frontend/liferay-portal/pull/1718#discussion_r763275783